### DISCUSSION TITLE
Question about resolution algorithm origin

### DISCUSSION THREAD

**Discussion by zsteig** [coursier/coursier#2674] [Q&A] [answered]
Is there any documentation regarding the resolution algorithm that Coursier uses, its structure, or how it compares to other dependency resolvers?

**Comment by julienrf** [comment]
There is some documentation here: https://get-coursier.io/docs/other-version-handling
Does that cover your need?

**Comment by eed3si9n** [comment, marked as answer]
Not a real documentation, but I've written a blog post related to this topic a while back - https://eed3si9n.com/dependency-resolver-semantics/
